cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -98,29 +91,6 @@ def test_schedule_crud(schedule_data, appliance):
     schedule.delete()
 
 
-@pytest.mark.rhel_testing
-@pytest.mark.sauce
-@pytest.mark.tier(3)
-@test_requirements.report
-def test_reports_disable_enable_schedule(schedule_data, appliance):
-    """
-    Polarion:
-        assignee: pvala
-        casecomponent: Reporting
-        caseimportance: high
-        initialEstimate: 1/10h
-    """
-    schedules = appliance.collections.schedules
-    schedule = schedules.create(**schedule_data)
-    schedules.disable_schedules(schedule)
-    assert not schedule.enabled
-    schedules.enable_schedules(schedule)
-    assert schedule.enabled
-    schedule.delete()
-
-
-@pytest.mark.sauce
-@pytest.mark.tier(3)
 @pytest.mark.meta(blockers=[BZ(1653796), BZ(1667064)])
 def test_menuwidget_crud(appliance):
```

Comment:
[1653796](https://bugzilla.redhat.com/show_bug.cgi?id=1653796) is verified. 
Take a look at it and remove from blockers.

Reply:
The test case doesn't belong to me, so it's not a good idea for me to remove the blockers, owner should get to decide that.